vncviewer/Viewport.cxx

```diff
@@ -551,10 +575,18 @@ int Viewport::handle(int event)
 {
   char *buffer;
   int ret;
+#if (defined(WIN32) || defined(__APPLE__))
   int buttonMask, wheelMask;
+#endif // !LINUX
   DownMap::const_iterator iter;
 
   switch (event) {
+#if ! (defined(WIN32) || defined(__APPLE__))
+  case FL_SHOW:
+    selectEvents();
+    break;
+#endif // LINUX
```

Comment:
Not sure I understand...

Reply:
The code you are writing is equally valid on Solaris or AIX. It's not Linux specific.

Reply:
Still confused :)

"Copy the `#if`"? Do you mean because there's two different variations, `!(WIN32 || APPLE)` and `!WIN32 && !APPLE`? Is it just the closing comment you're referring to when you say "X11", or do you mean that there's a check for X11?

Reply:
It's just the closing comment, yes. :)

It's definitely not just Linux, and for sanity it's usually best to verbatim copy the `#if` statement. I.e.:

```C
#ifdef FOO || BAR
#endif // FOO || BAR
```